doc/image_analysis.md

```diff
@@ -0,0 +1,89 @@
+# Image Analysis
+
+This feature allows users to upload FITS images from observations of a source, and then use STDPipe to **extract the photometry from the image**.
+As this feature requires additional system dependencies, we choose to make it optional.
+
+## Installing System Dependencies required by STDpipe (Debian-based Linux and WSL)
+
+```
+sudo apt install sextractor scamp psfex swarp
+```
+
+Also, one needs to install `snid`. To install it, you can run the following commands which will git clone and run an install script (works only on ubuntu for now, both arm64 and amd64)
+
+```
+git clone https://github.com/Theodlz/snid-install-ubuntu.git && \
+cd snid-install-ubuntu && sudo chmod +x install.sh && sudo bash ./install.sh
+```
+
+## Installing STDpipe
+
+As of right now, STDpipe is in development, and not available on pypi or conda yet.
+STDpipe is available at https://github.com/karpov-sv/stdpipe and is mirrored at https://gitlab.in2p3.fr/icare/stdpipe
+
+1. Get the installer
+
+```
+cd skyportal
+git clone https://github.com/karpov-sv/stdpipe.git
+```
+
+
+2. Installing it in your virtual environment
+
+```
+source skyportal_env/bin/activate
+cd stdpipe
+pip install -e .
+```
+
+## Configuration file
+
+By default, the app allows a maximum request size of 10MB (from the client). This is not enough for someone to upload most .fits files (compressed or not).
+To increase the maximum request size, change the `max_body_size: 10` parameter in the `config.yaml` file to a larger value, like `100` (in MB).
+
+Also, as mentioned earlier, you need to set `image_analysis` to `True` in the `config.yaml` file. By default, it is set to `False`.
```

Comment:
Great idea.

Reply:
However, we don't have tunable parameters to put in that config for now, what you should I do? Should I just put a dummy param like `enabled:true` to make sure that it's clear that it is a dict?

Reply:
I'm sure you can find something. E.g., photometric aperture radius. But there's no harm in using enable: true.

Reply:
Thanks for your answer guy. I'm changing it as you suggested. Now it just checks if `image_analysis`, has been added to the config, meaning later on it can be something other than `True` if need be